montydb/utils/io.py

```diff
@@ -71,7 +71,7 @@ def montyimport(database,
 def montyexport(database,
                 collection,
                 out,
-                fileds=None,
+                fields=None,
                 query=None,
                 json_options=None,
```

Comment:
Ouch, this is embarrassed. And the arg name in doc string is correctly spelled. :(
Have thought about adding `**kwargs` for backward compat but, just adding a change note in next release should be enough.